common/src/main/java/net/xolt/freecam/mixins/EntityRendererMixin.java

```diff
@@ -18,7 +18,7 @@ public class EntityRendererMixin {
 
     // Prevent rendering of nametag in inventory screen
     @Inject(method = "renderNameTag", at = @At("HEAD"), cancellable = true)
-    private void onRenderLabel(Entity entity, Component text, PoseStack poseStack, MultiBufferSource multiBufferSource, int light, float partialTick, CallbackInfo ci) {
+    private void onRenderLabel(EntityRenderState entityRenderState, Component component, PoseStack poseStack, MultiBufferSource multiBufferSource, int i, CallbackInfo ci) {
```

Comment:
```suggestion
    private void onRenderLabel(EntityRenderState renderState, Component component, PoseStack poseStack, MultiBufferSource multiBufferSource, int packedLightCoords, CallbackInfo ci) {
```

Looking at the parameter mappings for `Lnet/minecraft/client/gui/Font;drawInBatch` that `i` gets passed through to, we can use the name `packedLightCoords`.